plot.go

```diff
@@ -106,8 +106,10 @@ func New() (*Plot, error) {
 		Legend:          legend,
 	}
 	p.Title.TextStyle = draw.TextStyle{
-		Color: color.Black,
-		Font:  titleFont,
+		Color:  color.Black,
+		Font:   titleFont,
+		XAlign: -0.5,
```

Comment:
I like the idea of making it more obvious what these things do, but the functions also seemed a little difficult to wrap one's head around so I'm trying out a constant-based approach. Let me know what you think.